up-l1/mqtt_5.adoc

```diff
@@ -179,8 +179,8 @@ Shows how source and sink filters can be used to construct wildcard topics for l
 |===
 | goal | source filter | sink filter | listener topic
 
-| Subscribe to all publish messages from devices | empty | None | d/\+/+/\+/+////
-| Subscribe to all messages sent from a UAuthority | //vehicle_1.veh.com/FFFF/FF/FFFF | empty | d/vehicle_1.veh.com/\+/+/\+/+/\+/+/\+/+
+| Subscribe to all publish messages from devices | empty | None | d/\+/+/\+/+
+| Subscribe to all message types but publish messages sent from a UAuthority | //vehicle_1.veh.com/FFFF/FF/FFFF | empty | d/vehicle_1.veh.com/\+/+/\+/+/\+/+/\+/+
```

Comment:
what is the difference between a sinkFilter of "none" and sinkFilter of "empty"?

if this "empty" sinkFilter is a sinkFilter with
```
authorityName = "*"
uEid=0xffff
majorVersion = 0xff
resourceId = 0xffff
```
could we rename it with "any" or "wildcard"?